index.html

```diff
@@ -18,68 +18,81 @@
       <header>
         <nav>
           <a href="#"
-            ><img class="page--logo" src="images/Logo.png" alt=""
+            ><img class="page--logo" src="images/Logo.png" alt="Website logo"
           /></a>
           <a href="#"
-            ><img class="page--menu" src="images/menu.png" alt=""
+            ><img class="page--menu" src="images/menu.png" alt="The menu icon"
           /></a>
+          <ul class="nav-list">
+            <li class="nav-list-item"><a href="#">Portfolio</a></li>
+            <li class="nav-list-item"><a href="#">About</a></li>
+            <li class="nav-list-item"><a href="#">Contact</a></li>
```

Comment:
- [ ] Kindly try to match the text color to the Figma design. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![1](https://user-images.githubusercontent.com/70483835/221170383-20e34479-7d12-4c50-a729-2e20a3a321e1.png) |  ![1 2](https://user-images.githubusercontent.com/70483835/221170415-fd43fdd7-fd1c-49ed-a2f1-b18c498082ed.png)